server/main_chatbot.py

```
@@ -365,21 +365,34 @@ def save_incident_thought(content: str, force: bool = False):
+                                        (incident_id, now, cleaned_text, "analysis")
                                     )
+                                    set_rls_context(cursor, conn, user_id, log_prefix="[BackgroundChat]")
+                                    cursor.execute(
+                                        "UPDATE incidents SET updated_at = %s WHERE id = %s",
+                                        (now, incident_id),
+                                    )
+                                    incident_touched = cursor.rowcount == 1
                                 conn.commit()
-                                logger.info(f"[BackgroundChat] Saved thought for incident {incident_id}: {len(cleaned_text)} chars (incremental save)")
+                                if incident_touched:
+                                    logger.info(f"[BackgroundChat] Saved thought for incident {incident_id}: {len(cleaned_text)} chars (incremental save)")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)